extension/persistence/relational-jdbc/src/main/java/org/apache/polaris/extension/persistence/relational/jdbc/JdbcBasePersistenceImpl.java

```diff
@@ -143,12 +138,12 @@ private void persistEntity(
       @Nonnull PolarisCallContext callCtx,
       @Nonnull PolarisBaseEntity entity,
       PolarisBaseEntity originalEntity,
-      Statement statement)
+      CheckedFunction<String, Integer> fn)
```

Comment:
Do we really need such a generic type here? Could this be `updateFunction`?

Reply:
Are you suggesting making this always String, Integer ? or rename fn to updateFunction, I didn't made the name explicit updateFunction as we do both insert and update :P all DMLs. is kind of done statement.exceuteUpdate

Reply:
I see. I didn't realize it's used for the different functions in `Statement`. Fundamentally I guess it's something like:

```
public class QueryAction {
  /** Returns the number of affected rows */
  public int apply(String statement);
}
```

But using a @FunctionalInterface makes sense. For the name, maybe we can just call it as `queryConsumer` or `queryAction` or something to make it clear